docs/src/pages/_document.page.tsx

```diff
@@ -48,7 +48,9 @@ const getCSPContent = (context: Readonly<HtmlProps>) => {
     connect-src 'self' *.shortbread.aws.dev ${[
       ...ANALYTICS_CSP.all.connect,
       ...ANALYTICS_CSP.prod.connect,
-    ].join(' ')} https://*.algolia.net https://*.algolianet.com;
+    ].join(
+      ' '
+    )} https://*.algolia.net https://*.algolianet.com https://cdn.jsdelivr.net https://tfhub.dev https://storage.googleapis.com;
```

Comment:
Can we please review this change with AppSec since it adds cdn.jsdelivr.net?

Reply:
scheduled a meeting with ammar to discuss